bpo-45757: make dis reset extended_arg when op does not have an arg (#29480)
python/cpython: Misc/NEWS.d/next/Library/2021-11-08-23-22-14.bpo-45757.MHZHt3.rst

> Fix bug where :mod:`dis` produced an incorrect oparg when ``EXTENDED_ARG`` is followed by an opcode that does not have an arg.

Reviewer: ```suggestion
Fix bug where :mod:`dis` produced an incorrect oparg when :opcode:`EXTENDED_ARG` is followed by an opcode that does not use its argument.
```
